ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/push-queries.json

```diff
@@ -624,6 +704,29 @@
         ]}
       ]
     },
+    {
+      "name": "key, ROWPARTITION, ROWOFFSET in projection",
```

Comment:
I don't think this test adds anything beyond "explicit ROWPARTITION and ROWOFFSET" above. Let's remove it since RQTTs are expense.